### ISSUE TITLE
the output isnt working well for me,,,

### ISSUE THREAD

**Issue by petroi-ca** [ardha27/AICoverGen-NoUI-Colab#4]
so i tried generating my vocals using the audio path and but whenever its done generating the result is just a quieter version of the audio i used?? is it because my audio is an acapella??

**Comment by ardha27** [comment]
There are several file inside output folder, which one that you download?

**Comment by petroi-ca** [comment]
i download the one thats named “song (model Ver)”-

**Comment by ardha27** [comment]
Send me the screenshot of what input you put on all form

**Comment by petroi-ca** [comment]
![IMG_6651](https://github.com/ardha27/AICoverGen-NoUI-Colab/assets/145718480/37f9281d-b159-4ddb-9eef-90f4864758c2)
![IMG_6652](https://github.com/ardha27/AICoverGen-NoUI-Colab/assets/145718480/36867012-9d4c-4839-b5cc-1f3e9945dfe4)

**Comment by ardha27** [comment]
Did you try to use song instead?
Try to use pitch change all = 0

Just want to know, is it working properly on default setting?

**Comment by petroi-ca** [comment]
yeah it works properly with the default settings even when i use the audio path

**Comment by ardha27** [comment]
Try to use pitch change instead of change all?

**Comment by petroi-ca** [comment]
i used pitch change and it turned the generated vocals sort of glitchy and it didnt sound like that when i used pitch_change_all- i also tried it without the reverb settings enabled and it didnt generate the vocals at all, its just a quieter version of my audio file,,, is it okay if i send the audio that was generated?

**Comment by ardha27** [comment]
Do you use the same audio file when you try with default setting?

**Comment by petroi-ca** [comment]
yeah i used my audio file for converting

**Comment by ardha27** [comment]
Change the parameter one by one, find which one make your audio quiter

**Comment by petroi-ca** [comment]
i cant really tell whats making it quiet- i think it may be the reverb like if i turn it off the output doesnt work properly, but even then theres also a problem with the pitch_change for me and only pitch_change_all actually changes the pitch

**Comment by petroi-ca** [comment]
uhh heres what they sound like when i generate 

https://github.com/ardha27/AICoverGen-NoUI-Colab/assets/145718480/6702c4bd-2243-465f-92f1-d73ddda54da5

**Comment by ardha27** [comment]
1 in pitch change is equal to 1 octave or 12 pitch. Try to zero reverb parameter one by one, since i never bring reverb to zero
